Makefile

```diff
@@ -1,14 +1,16 @@
-.PHONY: build pod manifest manifest-create %-backend %-frontend
+.PHONY: build run pod manifest manifest-create %-backend %-frontend
 
 GROCY_VERSION = v3.1.1
 COMPOSER_VERSION = 2.1.5
 COMPOSER_CHECKSUM = be95557cc36eeb82da0f4340a469bad56b57f742d2891892dcb2f8b0179790ec
-IMAGE_COMMIT := $(shell git rev-parse --short HEAD)
-IMAGE_TAG := $(strip $(if $(shell git status --porcelain --untracked-files=no), "${IMAGE_COMMIT}-dirty", "${IMAGE_COMMIT}"))
+IMAGE_TAG ?= $(shell git describe --tags --match 'v*' --dirty)
 
+IMAGE_PREFIX ?= docker.io/grocy
 PLATFORM ?= linux/386 linux/amd64 linux/arm/v6 linux/arm/v7 linux/arm64/v8 linux/ppc64le linux/s390x
 
-build: pod manifest
+build: manifest run
+
+run: pod
```

Comment:
I like the idea of separating out the `run` make target :+1: 

It makes me wonder whether we should also remove `run` from the list of `build` dependencies -- what do you reckon?

Reply:
When testing my changes I didn't find `build` as a combination of `manifest` and `run` very useful. Usually I would first run `make manifest` first to build and only run `make run` explicitly when I could be reasonably certain that the build image is actually something I want to deploy.

Now that I think about it, having `run` as a dependency of `build` even seems kinda dangerous. It means that `git checkout <commit-with-broken-settings>; make build` is enough to wipe you existing grocy deployment and replace it with broken containers, even though `make build` looks quite harmless.

So, yeah, I think its a good idea to remove `run` from the dependencies of `build`.

On a related note: I recently used my changed version of the Makefile to deploy grocy productively in podman. When doing so I noticed that whenever I ran `make run` I usually wanted it to just create the containers (i.e. `podman create`)  and not start them immediately, because I usually wanted to configure other things (such as systemd units) first before starting. Maybe we should provide a `create` target which would use `podman create` to create the containers. We could then define `run` with a dependency on `create` and use `podman pod start` to start the pod. What do you think?

Reply:
@Kritzefitz Ok, agreed; let's move `run` separate to the `build` target.

Regarding a separate `create` target: that sounds reasonable to me.  Would the `create` target replace and/or include the steps that are currently inside the `pod` target?

Bubbling the generated container IDs up through the makefile targets could be a little tricky, but there are probably a few approaches available there.

Reply:
>  Would the create target replace and/or include the steps that are currently inside the pod target?

My current plan looks somewhat like this:

```makefile
create: pod
  podman create <arguments currently in run>
  podman create <arguments currently in run>

run: create
  podman pod start grocy-pod
```

i.e. the `pod` target would stay the same and `create` would include it.

> Bubbling the generated container IDs up through the makefile targets could be a little tricky, but there are probably a few approaches available there.

I don't think we need to. `podman pod start` can start the pod by name without knowing the container ids involved.

Reply:
All looks like a good plan to me :+1: 

Would you like write access to `grocy/grocy-docker` alongside these changes?  I'm happy to review them, and then you could merge them after that.

Reply:
Alright, I implemented the changes as discussed. Heads up: I force pushed my branch, so you might have to do `git reset --hard` on my branch to get the current version.

> Would you like write access to grocy/grocy-docker alongside these changes? I'm happy to review them, and then you could merge them after that.

That probably makes sense. Do you have any set guidelines or workflows I should follows when I get write access?

Reply:
**Review**
Thanks for the heads-up re: the force-push.  96ce409 is looking good from local testing here :+1: 

One thought: I wonder whether we should add a `stop` make target corresponding to the `run` target.

**Guidelines**
> Do you have any set guidelines or workflows I should follows when I get write access?

The main workflow-related item to understand, I think, is the release process.

Much of this is convention I've copied from elsewhere, and it's open to modification and/or automation:

Performing a release is aided by a [GitHub Actions configuration](https://github.com/grocy/grocy-docker/blob/5cecb649454d713cc3223755ea02fde5beba09eb/.github/workflows/build-docker.yml) that is triggered when a tagged [release](https://github.com/grocy/grocy-docker/releases) is created within this GitHub repository.  A successful build pushes the resulting images to Docker Hub with the corresponding tag.

During maintenance and review, each merged pull request should have an associated changelog entry added (usually under an 'Unreleased' section heading with no date -- like [this example](https://github.com/grocy/grocy-docker/commit/572cfceed0a38b3e7fb3378799710b9eea53e09e)).

There are a bunch of manual steps to prepare for and perform a release:

**Prerelease**
- Check for any updates to the `package.json` dependencies (currently, the only dep is a vulnerability scanner)
- Build all of the changes that are pending to be released (using the `Makefile` and/or `docker-compose`)
- Use `npm test` to perform automated testing (currently a build and then vulnerability scan)
- Perform some smoke testing; launch the application and ensure that it's possible to login and use the app

**Release**
- Replace all instances of the last-released version number (for example, in the [`docker-compose.yml`](https://github.com/grocy/grocy-docker/blob/5cecb649454d713cc3223755ea02fde5beba09eb/docker-compose.yml#L9) and [`package.json`](https://github.com/grocy/grocy-docker/blob/5cecb649454d713cc3223755ea02fde5beba09eb/package.json#L3) files) with the target release version
- Update the changelog with the target release version and release date
- Perform a `git commit` and then a `git tag` to label the release
- Push to GitHub, and create a new release based on the commit + tag, using the tag as the release name, and the most recent changelog section as the release description

**NB**: In practice I usually forget at least a few of these steps during releases, because I haven't formalized them as a checklist and/or automated them all.  Issue #109 also tracks a few ideas for areas of release improvement.

Reply:
> One thought: I wonder whether we should add a stop make target corresponding to the run target.

That would make sense for symmetry. But I'm not sure if we want to go that route. If we added `stop` it would merely be an alias for `podman pod stop grocy-pod`, without adding any functionality on top. So I don't see any value in being able to use `make stop` over `podman pod stop grocy-pod`.